test/blackbox-tests/test-cases/install-glob/install-glob-recursive.t

```diff
@@ -0,0 +1,69 @@
+Test that glob_files_rec can recursively find files
```

Comment:
```suggestion
Test that ``glob_files_rec`` can recursively find files
```

Reply:
For source code quoted in comments in .t files, I've switched to single backticks since that seems to be the convention.